proto/sentry_protos/snuba/v1/endpoint_get_traces.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "sentry_protos/snuba/v1/request_common.proto";
+import "sentry_protos/snuba/v1/trace_item_attribute.proto";
+import "sentry_protos/snuba/v1/trace_item_filter.proto";
+
+message TraceColumn {
+  enum Name {
+    NAME_UNSPECIFIED = 0;
+    NAME_TRACE_ID = 1;
+    NAME_START_TIMESTAMP = 2;
+    NAME_ROOT_SPAN_NAME = 3;
+    NAME_TOTAL_SPAN_COUNT = 4;
+    NAME_FILTERED_SPAN_COUNT = 5;
+  }
+
+  Name name = 1;
+  AttributeKey.Type type = 2;
+}
+
+message GetTracesRequest {
+  message OrderBy {
+    TraceColumn column = 1;
+    bool descending = 2;
+  }
+
+  RequestMeta meta = 1;
+  TraceItemFilter filter = 2;
+  PageToken page_token = 3;
+  uint32 limit = 4;
+
+  repeated OrderBy order_by = 5;
+  repeated TraceColumn columns = 6;
+}
+
+message GetTracesResponse {
+  message Trace {
```

Comment:
this type is also unclear to me - why does a trace have columns?

Reply:
and why doesn't it have a UUID?

Reply:
It's just a mechanism to only get back what you asked for. I defined a `enum` with the fields you can request from this API and the `columns` field is a list of the values you requested.

One of the value you can request is `GetTraces.Column.NAME_TRACE_ID` which will return a UUID of the `trace_id`.